core/contracts/oracle/implementation/Voting.sol

```diff
@@ -120,13 +119,17 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
     // Reference to the Finder.
     FinderInterface private finder;
 
-    // If non-zero, this contract has been migrated to this address. All voters and financial contracts should query the
-    // new address only.
+    // If non-zero, this contract has been migrated to this address. All voters and
+    //financial contracts should query the new address only.
```

Comment:
nit: space between "//" and "financial". Surprised that linter did not catch this